app/views/search/index.html.erb

```diff
@@ -179,7 +189,15 @@
               <li><%= snippet %></li>
             <% end %>
           </ol>
-        <span class="author"><%= format_time(e.event_datetime) %></span></dd>
+          <span class="last-modified">
+            <i class="fas fa-pen"></i>:
```

Comment:
We may not need `:` here.
Could you share screenshot?

Reply:
Sure!
I made a change thinking it would be more user-friendly, but do you think it's better without it?

<img width="1009" alt="Screenshot 2023-12-28 at 14 55 06" src="https://github.com/clear-code/redmine_full_text_search/assets/45173523/afed02cf-c47b-47a6-a1f4-8517baa2b237">

Reply:
Thanks.
We may need a separator by CSS (something like  " :pen:: 12/26/2023 05:50 PM | :calendar:: 12/26/2023 05:50 PM") between last modified time and registered time.
If we have a separator, I don't care with/without `:`.

Reply:
fix: [6f01fbb](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/6f01fbbfd28c641748d30236916fc06163c217e9)

Thank you for your suggestion. The visibility of content gets much better!
<img width="1004" alt="Screenshot 2023-12-28 at 17 45 39" src="https://github.com/clear-code/redmine_full_text_search/assets/45173523/824249b6-26d8-4f32-9e56-b33e5194a868">